### DISCUSSION TITLE
Classes with @Path annotation in the interface not being included

### DISCUSSION THREAD

**Discussion by cboldon** [smallrye/smallrye-open-api#1406] [Q&A]
Having just integrated open-api into a standalone service, I've noticed what I perceive as unexpected behavior.

A test resource with no interface gets picked up just fine, but the test resource I have with the @Path annotations on the interface does not.

Is there a configuration I'm missing, or is this not supported?

**Comment by MikeEdgar** [comment]
Interfaces without implementations will not be scanned unless you explicitly include them in the configuration scan classes/scan packages.

**Reply by cboldon** [reply]
Sorry, I don't think I explained it well. I have an implementation that implements an interface. The annotations are on the interface and not on the implementation. My expectation is what I am used to with swagger in the past; the implementation would be picked up by.

**Reply by MikeEdgar** [reply]
Are you adding both classes to the Jandex index before calling the scanner? If you have the project available on Github I can take a look.
